package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ckeditor-dev",
-	"version": "4.11.4.3",
+	"version": "4.11.4.4",
```

Comment:
can you also update https://github.com/interviewstreet/ckeditor-dev/blob/master/CHANGES.md so we have track of changes